package academy.devdojo.maratonajava.javacore.exception.test;

public class TestExceptionChecked01 {
    public static void main(String[] args) {
//        try {
//            FileReader.readFile("arquivo_inexistente.txt");
//        }
//        catch(IOException e) {
//            System.out.println("Erro ao ler o arquivo: "+ e.getMessage());
//        }

    }
}
